components/Hume/Expressions.tsx

```
@@ -0,0 +1,52 @@
+"use client";
+import { expressionColors, isExpressionColor } from "@/utils/expressionColors";
+import { expressionLabels } from "@/utils/expressionLabels";
+import { motion } from "framer-motion";
+import { CSSProperties } from "react";
+import * as R from "remeda";
+
+export default function Expressions({ values }: { values: Record<string, number> }) {
+  const top3 = R.pipe(
+    values,
+    R.entries(),
+    R.sortBy(([_, value]) => value),
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)